### ISSUE TITLE
python_cmd: no comma gives misleading error message about `{}`

### ISSUE THREAD

**Issue by cbm755** [cbm755/octsympy#304]
The command should always return a Tuple:
````
>> y = python_cmd( 'return S.One,');
````

Same without the comma.  Note misleading error message:
````
>> y = python_cmd( 'return S.One');
error: @sym/subsref: invalid subscript type '{}'
error: called from
    subsref at line 62 column 7
    python_cmd at line 144 column 3
error: evaluating argument list element number 1
error: called from
    python_cmd at line 144 column 3
>> 
````

**Comment by cbm755** [comment]
I cannot reproduce in 2.2.4: its still an error but not this confusing one.  Maybe this only happens after recent changes by @latot

**Comment by latot** [comment]
yes, sorry, i do this change before, remove this:
```
  if (~iscell(A))
    disp(A)
    % Python state undefined, so reset it (overkill for nostateful ipc)
    sympref reset
    error('OctSymPy:python_cmd:unexpected', 'python_cmd: unexpected return')
  end
```
if we don't send the las "," it returns a string, so i need remove it to test (don't is a cell), this was because some times when octsympy is broken we can find this type of messages, so i think if we can improve that part we can handle better the errors

**Comment by cbm755** [comment]
hmmm, ok well maybe there is room for improvement in `python_cmd` then.

That might be better than what I was planning: adding a "stub" error message for `{}` in `subsref`...

**Comment by latot** [comment]
if you put the same behavior to {} and ()?

**Comment by cbm755** [comment]
No I don't think we want that.  I will look into your comment above about `~iscell(A)`

**Comment by cbm755** [comment]
Fixed by #323, da2ff4da96ce801d1733cde4b9154572d14995e1
